phpunit/functional/HtmlTest.php

```diff
@@ -741,9 +741,9 @@ public function testAddConfirmationOnAction()
 
     public function testJsFunctions()
     {
-        $this->assertSame("$('#myid')", \Html::jsGetElementbyID('myid'));
-        $this->assertSame("$('#myid').trigger('setValue', 'myval');", \Html::jsSetDropdownValue('myid', 'myval'));
-        $this->assertSame("$('#myid').val()", \Html::jsGetDropdownValue('myid'));
+        $this->assertSame("$('#myid')", @\Html::jsGetElementbyID('myid'));
```

Comment:
Why a `@`?

Reply:
Because I deprecate them, but I can handle it differently.

Reply:
No, I see depreciation after adding comment, it's OK for the usage.